Pod/Classes/Plugin/Container/UIContainerPlugin.swift

```diff
@@ -16,6 +16,15 @@ public class UIContainerPlugin: UIPlugin, Plugin {
         super.init(frame: CGRectZero)
     }
     
+    public required init(context: UIBaseObject) {
+        super.init(frame: CGRectZero)
+        if let tempContainer = context as? Container {
```

Comment:
The `container` attribute should be an implicitly unwrapped optional, to avoid force unwrapping on calls and chaining `?`. We already check for `nil` instances on the initializer.
When using `if let` or `guard let` statements, we should keep the variable name clean of `temp` prefixes. Using just `container` as variable name would be better:
```swift
if let container = context as? Container {
    self.container = container
}
```
We should avoid using `;` as well, unless required.